nvflare/lighter/impl/signature.py

```diff
@@ -26,23 +27,31 @@ class SignatureBuilder(Builder):
     can be cryptographically verified to ensure any tampering is detected. This builder writes the signature.json file.
     """
 
-    def _do_sign(self, root_pri_key, dest_dir):
+    @staticmethod
+    def _do_sign(root_pri_key, dest_dir):
         signatures = sign_all(dest_dir, root_pri_key)
-        json.dump(signatures, open(os.path.join(dest_dir, "signature.json"), "wt"))
+        with open(os.path.join(dest_dir, ProvFileName.SIGNATURE_JSON), "wt") as f:
```

Comment:
some of the constants only used within a specific builder, do we still want to move them into the ProvFileName?

Reply:
It depends. If the constant is only used within the builder, we usually don't need to move. BUT for provisioned file names are moved into a central place to give us a holistic view of what files are generated by the provision. This could make it a little easier for maintenance.